scripts/mac_intel_install.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/zsh
+set -e
+# The purpose of the prefix is to give an indication of the point
+# of execution of this script, so that if something breaks it's easier
+# to see where that broke.
+prefix="[LM Install Script] "
+
+# Credit, Taken from: https://stackoverflow.com/a/34389425
+# Installs homebrew if it does not exist, or updates it if it does.
+which -s brew
+if [[ $? != 0 ]] ; then
+    echo "${prefix}Installing homebrew"
+    ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+else
+    echo "${prefix}Updating homebrew"
+    brew update
+fi
+
+echo "${prefix}Installing node"
+brew install node
+
+echo "${prefix}Installing node version manager (nvm)"
+brew install nvm
```

Comment:
can you follow this up with:

`nvm install v16.14.0`
`nvm alias default v16.14.0`
`nvm use default`

Reply:
Put this further down under the pyenv configuration